browser/brave_content_browser_client_unittest.cc

```diff
@@ -0,0 +1,43 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/brave_content_browser_client.h"
+
+#include "brave/common/extensions/extension_constants.h"
+#include "brave/components/brave_wallet/browser/buildflags/buildflags.h"
+#include "testing/gtest/include/gtest/gtest.h"
+
+typedef testing::Test BraveContentBrowserClientTest;
```

Comment:
nit: `using  BraveContentBrowserClientTest = testing::Test` is more preferred way in c++11 era :)